pkgs/development/python-modules/coinmetrics-api-client/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage, fetchPypi, lib, orjson, pandas, poetry, pythonOlder
+, python-dateutil, requests, typer, websocket-client }:
+buildPythonPackage rec {
+  pname = "coinmetrics-api-client";
+  version = "2022.8.12.7";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Gg6UVWaLrd/TrGVhVmUgH50nguqFa9qWXwNbGj2nmbI=";
+  };
+
+  nativeBuildInputs = [ poetry ];
```

Comment:
The package should use `poetry-core` if possible.

Fetch this patch: https://github.com/coinmetrics/api-client-python/pull/7/commits/2d79bb1272a0f28a136f5de8bdbfa6e090c37b30